Packs/CarbonBlackDefense/ReleaseNotes/4_0_0.md

```diff
@@ -0,0 +1,31 @@
+****
+#### Integrations
+
+##### New: Carbon Black Endpoint Standard v3
+
+- Deprecated the **cbd-find-events** and **cbd-find-events-results** commands, replaced by **cbd-find-observation**.
+- Deprecated the **cbd-find-events-details** and **cbd-find-events-details-results** commands, replaced by **cbd-find-observation-details**.
+- Deprecated the **cbd-get-policies** command, replaced by **cbd-get-policies-summary**.
```

Comment:
Deprecated the **cbd-get-policies** command, replaced with the **cbd-get-policies-summary** command.